elements/pfe-sass/functions/_functions.scss

```diff
@@ -178,10 +271,59 @@
 ///           background:  #{pfe-color(  surface--base )};
 ///         }
 ///         ```
-@function pfe-color($cssvar, $category: 'color', $map: $pfe-colors) {
-    @if $category != "" {
-      $category: "#{$category}--";
+@function pfe-color($cssvar,
+$category: 'color',
+$map: $pfe-colors) {
+    @if $category !="" {
+        $category: "#{$category}--";
     }
 
-    @return var( --#{$repo}-theme--#{$category}#{$cssvar}, #{map-get($map, $cssvar)});
+    @return var(--#{$repo}-theme--#{$category}#{$cssvar},
+    #{map-get($map, $cssvar)});
+}
+
+/// Local
+///
+/// Returns CSS Var for the local component-scoped variable
+///
+/// @param {String} $cssvar - Variable identifiers which are postfixed and combined using '--'
+/// @param {String} $region - Identifies the region or slot to which this is assigned
+/// @requires {String} $repo - Name of repo, which is "pfe"
+/// @requires {String} $LOCAL - Name of component, this is defined at the top of the sass file for the component
+/// @use -
+///     ```
+///     :host {
+///         padding-top:      #{pfe-local(paddingTop`)};
+///         padding-bottom:   #{pfe-local(paddingBottom)};
+///     }
+///     ```
+@function pfe-local($cssvar, $fallback: null, $region: null) {
+    // If a region value exists, build the region string
+    @if $region != null {
+        $region: "__#{$region}";
+    }
+    // Start building the variable declaration
+    $var-declaration: "--#{$repo}-#{$LOCAL}#{$region}--#{to-string($cssvar, '--')}";
+    // If a fallback is defined, pass that in too
+    @if $fallback != null {
+        $var-declaration: "#{$var-declaration}, #{$fallback}";
+    }
+    // Return the variable declaration string
+    @return var(#{$var-declaration});
 }
+
+/// Radio
+///
+/// Returns CSS Var for a broadcasted variable
+///
+/// @param {String} $cssvar - Variable identifiers which are postfixed and combined using '--'
+/// @requires {String} $repo - Name of repo, which is "pfe"
+/// @use -
+///     ```
+///     a:visited {
+///         color:       #{pfe-radio(  ui-link, visited  )};
```

Comment:
@castastrophe quick request, can you add 1 line here in the comments about what this would look like when compiled?

Reply:
Good idea, yes